modules/displayTodos.js

```diff
@@ -0,0 +1,85 @@
+import '../src/style.css';
+
+const DisplayTodos = () => {
+  let todos = JSON.parse(localStorage.getItem('todos')) || [];
+
+  const todoList = document.querySelector('#todo-list');
+  todoList.innerHTML = '';
+
+  todos
+    .sort((a, b) => a.index - b.index)
+    .forEach((todo) => {
+      const todoItem = document.createElement('div');
+      todoItem.classList.add('todo-item');
+
+      const label = document.createElement('label');
+
+      const input = document.createElement('input');
+      input.type = 'checkbox';
+      input.checked = todo.done;
+      label.appendChild(input);
+
+      const span = document.createElement('span');
+      span.classList.add('bubble');
+      label.appendChild(span);
+
+      const content = document.createElement('div');
+      content.classList.add('todo-content');
+      content.innerHTML = `<input type="text" value="${todo.content}" readonly>`;
```

Comment:
- [ ] **Optional:** Your todo's display undefined. The `content` attribute should be replaced with `description`.